docs/README.md

```diff
@@ -0,0 +1,3 @@
+## Almost there...
+
+Documentation for Most.js can be found at [mostcore.readthedocs.io](http://mostcore.readthedocs.io).
```

Comment:
Couple small requests: Let's change "Most.js" to "`@most/core`", and make the link `https`.  Thanks!

Reply:
Done.